pkg/render/fluentd.go

```diff
@@ -70,6 +70,11 @@ const (
 	SplunkFluentdSecretsVolName              = "splunk-certificates"
 	SplunkFluentdDefaultCertDir              = "/etc/ssl/splunk/"
 	SplunkFluentdDefaultCertPath             = SplunkFluentdDefaultCertDir + SplunkFluentdSecretCertificateKey
+	SysLogInternetCADir                      = "/etc/pki/tls/certs/"
```

Comment:
Perhaps "PublicCA" would be a more appropriate name than "InternetCA"

Reply:
Done